dephell_pythons/_shell_utils.py

```diff
@@ -0,0 +1,25 @@
+import os
+from pathlib import Path
+
+
+# https://stackoverflow.com/a/377028/8704691
+def is_file(path: Path) -> bool:
```

Comment:
If we check  not only that path exists and it's a file, but also that it is executable, I guess it's better to call it `is_executable` to avoid confusion
```suggestion
def is_executable(path: Path) -> bool:
```